src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22914.cs

```diff
@@ -0,0 +1,36 @@
+using System.Drawing;
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue22914 : _IssuesUITest
+	{
+		const string ButtonId = "Tap1Button";
+
+		readonly string[] _expectedNullBackgroundColorIds = [ButtonId, "ContentView1", "Label1", "VerticalStackLayout1"];
+		
+		public Issue22914(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "Setting BackgroundColor to null does not actually changes BackgroundColor";
+
+		[Test]
+		[Category(UITestCategories.ViewBaseTests)]
+		public void UpdateBackgroundColorToNull()
+		{
+			var tapAreaResult = App.WaitForElement(ButtonId, $"Timed out waiting for {ButtonId}");
+			tapAreaResult.Tap();
+
+			foreach (var elementId in _expectedNullBackgroundColorIds)
+			{
+				var element = App.WaitForElement(elementId, $"Timed out waiting for {elementId}");
+				var elementBackgroundColor = element.GetAttribute<Color?>("BackgroundColor");
```

Comment:
This is not implemented in appium by all the supported .NET MAUI platforms:

`System.NotImplementedException : 'BackgroundColor' attribute is unknown for the element. Only the following attributes are supported: [checkable, checked, {class,className}, clickable, {content-desc,contentDescription}, enabled, focusable, focused, {long-clickable,longClickable}, package, password, {resource-id,resourceId}, scrollable, selection-start, selection-end, selected, {text,name}, hint, extras, bounds, displayed, contentSize]`

As an alternative, you can use the VerifyScreenshot method. Will create a reference snapshot and the test will compare a new one created every time the test run with the reference one. In that way, can check the BackgroundColor. Other option, is create a Device Tests. Can find some samples accessing the platform control background color here https://github.com/dotnet/maui/blob/main/src/Controls/tests/DeviceTests/Elements/BoxView/BoxViewTests.cs#L29

Reply:
I think its safer to use VerifyScreenshot since we need to assert that property changes visually.
How do you take the screenshots for the test, in order to place them in snapshots folders.